modules/claims_api/app/views/claims_api/submission_report_mailer/submission_report.html.erb

```diff
@@ -10,6 +10,8 @@
       }
       table {
         border-collapse: collapse;
+        margin-bottom:35px;
+        min-width:475px;
```

Comment:
Not sure if this was stickly necessary, but it did look better to me with some margin between the tables and making them roughly all the same size.

Reply:
@rockwellwindsor-va what if we make the first column in each table the same size? Purely aesthetics, and not necessary, just a suggestion.

Reply:
@stiehlrod Ya I totally see that too, I'm just not sure it will work though given the consumer labels in the first table are long and the second column is short but then in the rest it is kind of flipped.  I want to make sure I do not set it and have blowouts or weird layout issues because of width sizing.  but ya these tables hurt my eyes, but without incorporating bootstrap and having them flex I just am not sure I can reliably get that dialed in.